docs/contents/components/layouts/divider/index.mdx

```diff
@@ -23,23 +23,25 @@ import { Divider } from "@yamada-ui/react"
 
 ```tsx
 <Flex direction="column" gap="md">
-  <Divider variant="solid" />
-
-  <Divider variant="dashed" />
-
-  <Divider variant="dotted" />
+  <For
+    each={[{ variant: "solid" }, { variant: "dashed" }, { variant: "dotted" }]}
+  >
+    {({ variant }, index) => <Divider key={index} variant={variant} />}
+  </For>
 </Flex>
 ```
 
 ### Draw a vertical dividing line
 
 ```tsx
 <Flex h="sm" gap="md">
-  <Divider orientation="vertical" variant="solid" />
-
-  <Divider orientation="vertical" variant="dashed" />
-
-  <Divider orientation="vertical" variant="dotted" />
+  <For
+    each={[{ variant: "solid" }, { variant: "dashed" }, { variant: "dotted" }]}
```

Comment:
https://github.com/yamada-ui/yamada-ui/pull/3838/files#r1875503601

This line should be modified in the same manner.